console/src/main.js

```diff
@@ -100,6 +234,9 @@ function openFileBrowser(path) {
     }
 }
 
+app.on('window-all-closed', function() {
+});
```

Comment:
We should probably make a note that this is needed to keep the application from closing.